import_export_extensions/resources.py

```diff
@@ -200,14 +199,18 @@ def export(
             queryset=queryset,
         )
         return super().export(  # type: ignore
-            *args,
-            **kwargs,
             queryset=queryset,
+            **kwargs,
         )
 
-    def export_resource(self, obj, fields: list[fields.Field] | None = None):
+    def export_resource(
+        self,
+        obj,
+        selected_fields: list[fields.Field] | None = None,
+        **kwargs,
+    ):
```

Comment:
This change is breaking change on our side, it's okay since we are still pre-release, but might need to update version here https://github.com/saritasa-nest/django-import-export-extensions/blob/6e02e184c3fee6cc615c4351d4d6fe79bd727d81/pyproject.toml#L41
Since if we try to use old version, it will break, probably